tests/unit_tests/blas/extensions/gemmt.cpp

```diff
@@ -119,9 +119,8 @@ int test(device* dev, oneapi::mkl::layout layout, oneapi::mkl::uplo upper_lower,
 #endif
     }
     catch (exception const& e) {
-        std::cout << "Caught synchronous SYCL exception during GEMMT:\n"
-                  << e.what() << std::endl
-                  << "OpenCL status: " << e.get_cl_code() << std::endl;
+        std::cout << "Caught synchronous SYCL exception during GEMMT:\n" << e.what() << std::endl;
```

Comment:
Let's keep the same format if possible.

Reply:
If I understand correctly you would like to have the `<< e.what() << std::endl;` in a new line, like for example in  tests/unit_tests/blas/batch/axpy_batch_stride.cpp, is that correct? Changing this consistently would entail changing 104 occasions and it wouldn't be clang-format conformant.

Reply:
Why wouldn't it be clang-format conformant? The code before your change was clang-format conformant. 
The concern here is, in case we want to do some global changes in all tests, we typically run some script and not having the same format across tests will break the script. Then, someone will have to go fix them.

Reply:
I think it is because we have made the error printing statement much shorter by printing the OpenCL status in the `print_error_code` function. This makes the line break necessary for only the functions with longer names, such as the batched functions. For all the other tests, there is no line break. 

Should we add the line break for every function, or rather delete the new line for the few longer lines (batched functions mainly)? I think either way the code won't adhere to the clang-format, but removing the new lines for the batched functions probably results in fewer errors.

Reply:
Thanks for looking at it in more detail. Based on your findings, i think we can try to handle this case in the script.